core/src/test/scala/org/apache/spark/deploy/history/FsHistoryProviderSuite.scala

```diff
@@ -291,6 +292,159 @@ class FsHistoryProviderSuite extends SparkFunSuite with BeforeAndAfter with Matc
     }
   }
 
+  test("Handling executor log url without renewing") {
```

Comment:
renewing?

I also dislike this style of test name. Every test name starts with the same prefix, which makes it hard to differentiate them.

To follow the pattern used in this class, for example:

```
test("log urls without customization")
test("custom log urls")
test("custom log urls with invalid attribute")
```

etc etc

If your test name is wrapping to the next line, it's too long.

Reply:
Actually I prefer grouping similar tests as same prefix as well as enough long explanation to make test name self-describing, but no strong opinion, and I also agree it may make hard to differentiate them. I'll go through test names and renaming them.